ui/app/AppLayouts/Chat/stores/CommunitiesStore.qml

```diff
@@ -7,6 +7,8 @@ import utils 1.0
 QtObject {
     id: root
 
+    property var mintingModuleInst: mintingModule
```

Comment:
We've had some discussions about redefining stores and try to just expose the explicit backend needs, so I guess we can remove this property and just only keep the model property. WDYT?

If it is useful to have this property for internal store use, you can add it as a `d` "private" property instead.